scripts/regolith-set-swaybg

```diff
@@ -17,21 +17,18 @@ fi
 echo "$wallpaper IS valid XML? $is_valid_xml"
 
 if [[ "$wallpaper" == "" || ! $is_valid_xml ]]; then
-	exec swaymsg output "*" background "$color" solid_color
+  exec swaymsg output "*" background "$color" solid_color
 fi
 
-if $is_valid_xml; then
+if [[ $is_valid_xml ]]; then
```

Comment:
This doesn't work. The control flow goes inside the XML branch, with this change.

Reply:
Right, my bash is a bit rusty, thanks.